Frameworks/CoreGraphics/CGDataProvider.mm

```diff
@@ -435,4 +448,8 @@ size_t _CGDataProviderGetSize(CGDataProviderRef provider) {
     return provider ? provider->GetSize() : 0;
 }
 
+CFURLRef _CGDataProviderGetURL(CGDataProviderRef provider) {
+    return provider ? provider->GetURL() : nullptr;
```

Comment:
This is consistent w/ the file, and is a case where a ternary is just as if not more clear than a macro IMO